pkg/storaged/fsys-tab.jsx

```diff
@@ -314,6 +316,16 @@
                 .then(utils.reload_systemd));
     }
 
+    let at_boot;
+    if (opt_never_auto)
+        at_boot = "never";
+    else if (opt_netdev)
+        at_boot = "netdev";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-142402-f4c856ed-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)